WordPress/WordPressShare/PostStatusPickerViewController.swift

```diff
@@ -0,0 +1,98 @@
+import Foundation
+import UIKit
+import WordPressShared
+
+class PostStatusPickerViewController : UITableViewController
+{
+    // MARK: - View Lifecycle
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        setupView()
+        setupTableView()
+        setupNoResultsView()
+        loadStatuses()
+    }
+    
+    
+    // MARK: - UITableView Methods
+    override func numberOfSectionsInTableView(tableView: UITableView) -> Int {
+        return 1
+    }
+    
+    override func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return statuses.count
+    }
+    
+    override func tableView(tableView: UITableView, heightForRowAtIndexPath indexPath: NSIndexPath) -> CGFloat {
+        return rowHeight
+    }
+    
+    override func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
+        var cell = tableView.dequeueReusableCellWithIdentifier(reuseIdentifier)
```

Comment:
WPReusableTableViewCell(s) detached in PR #4949, for easier review. I've pre-emptively merged that PR into this branch, and simplified Cell Reuse with @aerych's super awesome method.

Thank you both!